src/runner/bootstrapper.js

```diff
@@ -74,6 +74,9 @@ export default class Bootstrapper {
         if (!this.browsers.length)
             throw new Error(getText(MESSAGE.browserNotSet));
 
+        if (this.browsers.some(browser => browser instanceof BrowserConnection && browser.disconnected))
+            throw new Error(getText(MESSAGE.cantEstablishBrowserConnection));
```

Comment:
We need another message here. We don't try to establish connection, we have a connection passed which is already disconnected.